.devcontainer/ubuntu18.04/devcontainer.json

```diff
@@ -1,9 +1,38 @@
+// For format details, see https://aka.ms/devcontainer.json.
```

Comment:
I don't feel we should duplicate information into two files.

Reply:
Needed to quickly find documentation on `devcontainer.json`.